qa-tests-backend/src/main/groovy/services/NetworkGraphService.groovy

```diff
@@ -45,75 +43,50 @@ class NetworkGraphService extends BaseService {
         }
     }
 
+    @Retry
     static createNetworkEntity(String clusterId, String name, String cidr, Boolean isSystemGenerated) {
-        try {
-            if (clusterId == null) {
-                throw new RuntimeException("Cluster ID is required to create a network entity")
-            }
-            if (name == null) {
-                throw new RuntimeException("Name is required to create a network entity")
-            }
-            if (cidr == null) {
-                throw new RuntimeException("CIDR address needs to be defined to create a network entity")
-            }
-            // Create entity for request
-            ExternalSource.Builder entity =
-                    ExternalSource
-                            .newBuilder()
-                            .setName(name)
-                            .setCidr(cidr)
-                            .setDefault(isSystemGenerated)
-
-            // Create request
-            CreateNetworkEntityRequest request =
-                    CreateNetworkEntityRequest
-                            .newBuilder()
-                            .setClusterId(clusterId)
-                            .setEntity(entity)
-                            .build()
-
-            return getNetworkGraphClient().createExternalNetworkEntity(request)
-        } catch (Exception e) {
-            log.error("Exception while creating network entity", e)
+        if (clusterId == null) {
+            throw new RuntimeException("Cluster ID is required to create a network entity")
         }
-    }
-
-    static deleteNetworkEntity(String entityID) {
-        try {
-            // Create request
-            getNetworkGraphClient().deleteExternalNetworkEntity(ResourceByID.newBuilder().setId(entityID).build())
-        } catch (Exception e) {
-            log.error("Exception while deleting network entity", e)
+        if (name == null) {
+            throw new RuntimeException("Name is required to create a network entity")
+        }
+        if (cidr == null) {
+            throw new RuntimeException("CIDR address needs to be defined to create a network entity")
```

Comment:
Do we want to retry if the three required inputs are missing? 
Maybe we should only retry the block that starts below?

Reply:
I consider this resolved by [6066e02](https://github.com/stackrox/stackrox/pull/15310/commits/6066e028dd79e9222947a647a86c1889dff2dfb0)